File: engine/engine-model/src/main/scala/server/movement/ChangeEquip.scala
package server.movement

import java.awt.Point

import tools.data.output.LittleEndianWriter

class ChangeEquip(private var _wui: Int) extends LifeMovementFragment {
  override def serialize(lew: LittleEndianWriter): Unit = {
    lew.write(10)
    lew.write(_wui)
  }

  override def position(): Point = {
    new Point(0, 0)
  }
}

File: engine/engine-model/src/main/java/server/movement/ChangeEquip.java
package server.movement;

import java.awt.Point;

import tools.data.output.LittleEndianWriter;

public class ChangeEquip implements LifeMovementFragment {
   private final int wui;

   public ChangeEquip(int wui) {
      this.wui = wui;
   }

   @Override
   public void serialize(LittleEndianWriter lew) {
      lew.write(10);
      lew.write(wui);
   }

   @Override
   public Point position() {
      return new Point(0, 0);
   }
}
